### ISSUE TITLE
Unable to publish with AssumeRolePolicyDocument in Globals

### ISSUE THREAD

**Issue by jplock** [awslabs/serverless-application-model#1406]
Based on https://github.com/awslabs/serverless-application-model/pull/1311, it looks like i should be able to put `AssumeRolePolicyDocument` in `Globals`, but when trying to run `sam publish`, I'm getting:

```
Publish Failed
Error: Invalid Serverless Application Specification document. Number of errors found: 1. Errors: 'Globals' section is invalid. 'AssumeRolePolicyDocument' is not a supported property of 'Function'. Must be one of the following values - ['Handler', 'Runtime', 'CodeUri', 'DeadLetterQueue', 'Description', 'MemorySize', 'Timeout', 'VpcConfig', 'Environment', 'Tags', 'Tracing', 'KmsKeyArn', 'AutoPublishAlias', 'Layers', 'DeploymentPreference', 'PermissionsBoundary', 'ReservedConcurrentExecutions', 'ProvisionedConcurrencyConfig']
Please follow the instructions in https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html
```

I'm using the aws-sam-cli 0.40.0 installed via pip

`sam validate` says the template is fine

**Comment by keetonian** [comment]
@jplock I think that should work, I'll follow up with the Serverless Application Repository team to see what the issue is and let you know.

**Comment by jplock** [comment]
Thanks @keetonian.  Sorry for filing this bug in multiple places as I wasn't sure where to report it.  I also created it on https://github.com/awslabs/aws-sam-cli/issues/1745

**Comment by jplock** [comment]
Hi @keetonian any update on this?

**Comment by jplock** [comment]
Should be fixed in aws-sam-cli 0.41.0.  Will re-open if still an issue.
